### PR TITLE
debug screenings banner

### PR DESCRIPTION
**Pull request by SamirTouira**
Number of sessions modified (5 sessions max), select option reviewed and adapted and css a bit modified
Fix: #4183

### CODE REVIEW

File: libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.ts [hunk 1]
```diff
@@ -22,12 +22,19 @@ import { OrganizationService } from '@blockframes/organization/+state';
 })
 export class UpcomingScreeningsComponent {
 
-  public sessionCtrl = new FormControl('first');
+  public sessions = {
+    0: 'first',
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.ts:26]: I think you want to have a titlecase here `First`, 'Second', ...

File: libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.html [hunk 2]
```diff
@@ -2,31 +2,24 @@
...
-          </span>
-          <span class="mat-body-1" fxLayout="row" fxLayoutAlign="center start">
-            <!-- Start hour -->
-            {{ screening.start | date: 'h a' }}
-            <!-- Splitter -->
-            -
-            <!-- End Hour -->
-            {{ screening.end | date: 'h a'}}
-          </span>
+          <h5>{{ sessions[index] }} Session</h5>
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.html:7]: Either ` | titlecase` or apply the titlecase directly in the property "sessions" (see comment below)

File: libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.ts [hunk 3]
```diff
@@ -22,12 +22,19 @@ import { OrganizationService } from '@blockframes/organization/+state';
 })
 export class UpcomingScreeningsComponent {
 
-  public sessionCtrl = new FormControl('first');
+  public sessions = {
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.ts:25]: Use an array when you work with number index in order: `public sessions = ['First', 'Second', 'Thrid' ,...]`

**Review by GrandSchtroumpf — CHANGES_REQUESTED** [review]

File: libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.html (hunk 2 above)
Comment (SamirTouira) [reply libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.html:7]: We had apply the propety `text-transform: uppercase` to the <h5> for sessions[index] in the scss file, are you ok with that ?

File: libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.ts (hunk 1 above)
Comment (SamirTouira) [reply libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.ts:26]: The titlecase for here is already used in the upcoming-screenings.component.html : 

`     18   <mat-option
      19   *ngFor="let screening of screenings; let index = index"
      20    [value]="index">
      21   {{ sessions[index] | titlecase }} Session
      22   </mat-option>`

Is this correct ?

File: libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.html (hunk 2 above)
Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/components/upcoming-screenings/upcoming-screenings.component.html:7]: ok

**Review by GrandSchtroumpf — APPROVED** [review]

**Pull request merged**